routes/users/users.js

```diff
@@ -0,0 +1,62 @@
+const users = require('express').Router()
+
+let userArr = [
+    {username: "user1",
+    email: "user1@pursuit.org",
+    activationDate: "jan 1, 2017"},
+
+    {username: "user_two",
+    email: "user_two@pursuit.org",
+    activationDate: "jan 1, 2019"},
+
+    {username: "user3",
+    email: "user3@pursuit.org",
+    activationDate: "jan 32, 2021"},
+    
+    {username: "user1234",
+    email: "user1234@gmail.com",
+    activationDate: "January 1, 1970"}
+]
+users.get('/', (req,res) => {
+
+    res.json(userArr)
+})
+users.post('/',(req,res) => {
+    userArr.push(req.body)
+    res.json({
+        status:200,
+        message: "posted new user",
+        newUser: req.body
+    })
+
+})
+users.delete('/:id',(req,res) => {
+    res.json("deleted: " + req.params.id)
+})
+users.patch('/:id', (req,res) => {
+    userArr.push(req.body)
```

Comment:
Here you are not actually updating the user but rather adding new information. You'd need to find the user that matches that parameters, and then update it appropriately.